src/common/NavBar/index.js

```diff
@@ -0,0 +1,53 @@
+import React from "react";
+import { BrowserRouter as Router, Switch, Route, Link } from "react-router-dom";
+
+const NavBar = () => {
```

Comment:
Mobile navigation needs a state to be managed. Let's add that as well.
Refer to this: https://github.com/Suvink/magnate-shipping-landing/blob/main/src/common/NavBar/index.js

Reply:
Fixed with [a76f4af](https://github.com/Suvink/let-me-react/pull/15/commits/a76f4af6e4662e78f6b4c5ee4e0a47a630c237dd)